suite-common/message-system/config/config.v1.json

```diff
@@ -564,6 +564,127 @@
                 }
             }
         },
+        {
+            "conditions": [
+                {
+                    "environment": {
+                        "desktop": ">25.2.2",
+                        "mobile": "!",
+                        "web": ">25.2.2"
+                    },
+                    "devices": [
+                        {
+                            "model": "T1B1",
+                            "firmware": "1.12.1",
+                            "bootloader": "*",
+                            "variant": "*",
+                            "firmwareRevision": "*",
+                            "vendor": "*"
+                        },
+                        {
+                            "model": "1",
```

Comment:
This might be redundant, devices with firmware `1.12.1` probably always report as `T1B1`. But needs to be checked.

Reply:
This depends on Suite (Connect), not the FW, right? So I believe it's redundant here and T1B1 is enough, but better to test it :-D